README.md

```diff
@@ -496,6 +478,45 @@ wind.on('accelData', function(e) {
 });
 ````
 
+### Voice
+
+The `Voice` module allows you to interact with Pebble's dictation API on supported platforms (Basalt and Chalk).
+
+````js
+var Voice = require('ui/voice');
+````
+
+#### Voice.dictate('start', [confirmDialog,] callback)
+
+This function starts the dictation UI, and invokes the callback upon completion. The callback is invoked with an event with the following fields:
```

Comment:
These sentences are awkward. How about:

"This function starts the dictation system UI and invokes the callback upon completion. The following callback parameters are provided to the caller:'